BlackJackets API/src/main/java/BlackJackets/BlackJackets/service/impl/MessageServiceImpl.java

```diff
@@ -0,0 +1,58 @@
+package BlackJackets.BlackJackets.service.impl;
+
+import BlackJackets.BlackJackets.data.MessageRepo;
+import BlackJackets.BlackJackets.data.VenueRepo;
+import BlackJackets.BlackJackets.dto.MessageDTO;
+import BlackJackets.BlackJackets.models.Message;
+import BlackJackets.BlackJackets.models.Venue;
+import BlackJackets.BlackJackets.service.MessageService;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+public class MessageServiceImpl implements MessageService {
+
+    @Autowired
+    private ModelMapper modelMapper;
+
+    @Autowired
+    private MessageRepo messageRepo;
+
+    @Autowired
+    private VenueRepo venueRepo;
+
+    // Create Message
+    @Override
+    public String createNewMessage(MessageDTO messageDTO, int venueId) {
+        Message message = this.modelMapper.map(messageDTO, Message.class);
+        Optional<Venue> venue = venueRepo.findById(venueId);
+        message.setVenue(venue.get());
+        this.messageRepo.save(message);
+        return "Message added successfully";
+    }
+
+    // Get All Messages
+    @Override
+    public List<MessageDTO> getAllMessages() {
```

Comment:
I am not sure why this method returns a `List<MessageDTO>` while the other returns `List<Message>`, but they both seem to work. I think if we want a future option for deleting these then it might need to be changed but that can be a future change